wbooks/src/app/components/BackIcon/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Image } from 'react-native';
+import { IMAGES } from '@constants/images';
```

Comment:
We import the asset directly, like:

```suggestion
import icBack from '@assets/icBack.png';
```